src/styles/calendar.scss

```diff
@@ -0,0 +1,57 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+  width: calc((#{$day-size} * #{$columns}) + (#{$gap} * #{$columns - 1}));
+  background-color: #fff;
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    background-color: #eee;
+    display: flex;
+    border: 1px solid #000;
+    box-sizing: border-box;
+    transition:
+      background-color 0.5s,
+      transform 0.5s;
+
+    &::before {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      margin: auto;
+      font-family: Arial, sans-serif;
+      font-size: 30px;
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      transition:
+        background-color 0.5s,
+        transform 0.5s;
```

Comment:
The 'transition' property here is redundant because it is already defined in the '&__day' selector. Consider removing it to avoid duplication.

Reply:
> The 'transition' property here is redundant because it is already defined in the '&__day' selector. Consider removing it to avoid duplication.

Good point